src/python/sdk/econia_sdk/view/market.py

```diff
@@ -140,6 +218,18 @@ def get_open_orders(
     n_asks_max: int = _HI_64,
     n_bids_max: int = _HI_64,
 ) -> dict:
+    """
+    Index order book for given market ID into "asks" and "bids"
+    vectors. Vectors sorted by price-time priority.
+
+    Parameters
```

Comment:
typo: `Parameters:` per https://github.com/econia-labs/econia/pull/394#pullrequestreview-1555321875

Reply:
And similarly for all new docstrings